File: composeApp/src/androidMain/kotlin/com/kmp/dardev/league/app/template/presentation/screen/parameter/component/ParameterInfoSection.kt
package com.kmp.dardev.league.app.template.presentation.screen.parameter.component

import androidx.compose.foundation.background
import androidx.compose.foundation.clickable
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.shape.CircleShape
import androidx.compose.material3.*
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.platform.UriHandler
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.text.style.TextDecoration
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp

@Composable
fun ParameterInfoSection(uriHandler: UriHandler) {
    Column(
        modifier = Modifier.fillMaxSize().padding(16.dp),
        horizontalAlignment = Alignment.CenterHorizontally,
    ) {
        Box(
            modifier =
                Modifier
                    .size(150.dp)
                    .clip(CircleShape)
                    .background(Color(0xFFFFF9C4)),
            contentAlignment = Alignment.Center,
        ) {
            Text(
                text = "Logo App",
                fontSize = 24.sp,
                fontWeight = FontWeight.Bold,
            )
        }

        Spacer(modifier = Modifier.height(8.dp))

        Text(
            text = "Version 0.0.1",
            fontSize = 16.sp,
        )

        Text(
            text = "Sortie le 2 février 2025",
            fontSize = 16.sp,
        )

        Spacer(modifier = Modifier.height(16.dp))

        Text(
            text = "About us",
            color = MaterialTheme.colorScheme.primary,
            fontSize = 18.sp,
            fontWeight = FontWeight.Bold,
        )

        Text(
            text = "Lorem ipsum lorem ipsum lorem ipsum",
            textAlign = TextAlign.Center,
            modifier = Modifier.padding(horizontal = 32.dp),
        )

        Spacer(modifier = Modifier.height(8.dp))

        Row(
            verticalAlignment = Alignment.CenterVertically,
        ) {
            Text(text = "Contact -> ")
            Text(
                text = "contact@email.com",
                color = MaterialTheme.colorScheme.primary,
                modifier =
                    Modifier.clickable {
                        uriHandler.openUri("mailto:contact@email.com")
                    },
            )
        }

        Row(
            verticalAlignment = Alignment.CenterVertically,
        ) {
            Text(text = "Feedback -> ")
            Text(
                text = "google forms",
                color = MaterialTheme.colorScheme.primary,
                modifier =
                    Modifier.clickable {
                        uriHandler.openUri("https://forms.google.com")
                    },
            )
        }

        Row(
            verticalAlignment = Alignment.CenterVertically,
        ) {
            Text(text = "Privacy -> ")
            Row(
                verticalAlignment = Alignment.CenterVertically,
            ) {
                Text(
                    text = "https://my-site.com",
                    color = MaterialTheme.colorScheme.primary,
                    textDecoration = TextDecoration.Underline,
                    modifier =
                        Modifier.clickable {
                            uriHandler.openUri("https://my-site.com")
                        },
                )
            }
        }
    }
}

File: iosApp/iosApp/presentation/screens/Parameter/Components/ParameterInfoSection.swift
//
//  ParameterInfoSection.swift
//  iosApp
//
//  Created by Darren on 2025-10-12.
//  Copyright © 2025 orgName. All rights reserved.
//

import SwiftUI

struct ParameterInfoSection: View {
    var body: some View {
        VStack(alignment: .center, spacing: 12) {
            ZStack {
                Circle()
                    .fill(Color.yellow.opacity(0.4))
                    .frame(width: 150, height: 150)
                Text("Logo App")
                    .font(.system(size: 24, weight: .bold))
            }

            Text("Version 0.0.1")
                .font(.system(size: 16))

            Text("Sortie le 2 février 2025")
                .font(.system(size: 16))

            Spacer().frame(height: 16)

            Text("About us")
                .font(.system(size: 18, weight: .bold))
                .foregroundColor(.blue)

            Text("Lorem ipsum lorem ipsum lorem ipsum")
                .multilineTextAlignment(.center)
                .padding(.horizontal, 32)

            VStack(alignment: .leading, spacing: 8) {
                HStack {
                    Text("Contact ->")
                    Link("contact@email.com", destination: URL(string: "mailto:contact@email.com")!)
                        .foregroundColor(.blue)
                }

                HStack {
                    Text("Feedback ->")
                    Link("google forms", destination: URL(string: "https://forms.google.com")!)
                        .foregroundColor(.blue)
                }

                HStack {
                    Text("Privacy ->")
                    Link("https://my-site.com", destination: URL(string: "https://my-site.com")!)
                        .foregroundColor(.blue)
                }
            }
            .padding(.top, 8)
        }
        .padding(16)
        .frame(maxWidth: .infinity, alignment: .center)
    }
}
